app/controllers/jobs_controller.rb

```diff
@@ -41,8 +29,8 @@ def index
     end
 
     # Get the complete lists of live and dead jobs from the server
-    raw_live_jobs = getCurrentJobs
-    raw_dead_jobs = getDeadJobs
+    raw_live_jobs = TangoClient.tango_jobs()["jobs"]
```

Comment:
I think it's a good idea to acknowledge error cases. Null would be good, but how about raising exceptions with an explanatory text? That way we can just do begin/rescue and also display human readable errors.